custom/enikshay/expressions.py

```diff
@@ -296,16 +296,20 @@ def configure(self, person_id_expression):
 
     def __call__(self, item, context=None):
         person_id = self._person_id_expression(item, context)
-        domain = context.root_doc['domain']
         if not person_id:
             return None
+        return self._get_open_episode_case_json(person_id, context)
+
+    @staticmethod
```

Comment:
Does the ucr_context_cahce decorator require these to be staticmethod?

Reply:
Nope, they could also be normal instance methods or module-level functions.